packages/headless/src/api/commerce/common/response.ts

```diff
@@ -7,11 +7,19 @@ import {Pagination} from './pagination';
 import {Product} from './product';
 import {Sort} from './sort';
 
-export interface CommerceSuccessResponse {
+interface BaseCommerceSuccessResponse {
   responseId: string;
   products: Product[];
-  facets: AnyFacetResponse[];
   pagination: Pagination;
+}
+
+export interface RecommendationsCommerceSuccessResponse
+  extends BaseCommerceSuccessResponse {
+  headline: string;
+}
```

Comment:
This type should reside somewhere like `api/commerce/recommendations/response.ts`
https://github.com/coveo/ui-kit/tree/c71140b51b118e9516aa8e97316ecf0801ac9827/packages/headless/src/api/commerce/recommendations